### PR TITLE
Whitelist Kick

### PR DESCRIPTION
**Pull request by hamperfait**
Whitelist kick is like somekick, but works in reverse.
Those hosts that are written will NOT be kicked out, the rest will be.

### CODE REVIEW

File: kickthemout.py [hunk 1]
```diff
@@ -89,6 +89,8 @@ def optionBanner():
     sleep(0.2)
     print('\t{}[{}3{}]{} Kick ALL Off'.format(YELLOW, RED, YELLOW, WHITE))
     sleep(0.2)
+    print('\t{}[{}4{}]{} Kick ALL BUT SOME Off (whitelist)'.format(YELLOW, RED, YELLOW, WHITE))
```
Comment (k4m4) [inline kickthemout.py]: `Kick ALL BUT SOME Off (whitelist)` --> `Kick ALL EXCEPT WHITELIST Off`

File: kickthemout.py [hunk 2]
```diff
@@ -670,6 +766,9 @@ def main():
                 elif choice == '3':
                     kickalloff()
 
+                elif choice == '4':
+                    kickwhitelistoff()
```
Comment (k4m4) [inline kickthemout.py]: `kickwhitelistoff` --> `kickexceptoff`

File: kickthemout.py [hunk 3]
```diff
@@ -544,6 +546,100 @@ def kicksomeoff():
         print("{}Re-arped{} targets successfully.{}".format(RED, GREEN, END))
 
 
+# kick whitelisting devices
+def kickwhitelistoff():
+    os.system("clear||cls")
+
+    print("\n{}kickWHITELISTOff{} selected...{}\n".format(RED, GREEN, END))
```
Comment (k4m4) [inline kickthemout.py]: `kickWHITELISTOff ` --> `kickEXCEPTOff`

File: kickthemout.py [hunk 4]
```diff
@@ -544,6 +546,100 @@ def kicksomeoff():
         print("{}Re-arped{} targets successfully.{}".format(RED, GREEN, END))
 
 
+# kick whitelisting devices
+def kickwhitelistoff():
```
Comment (k4m4) [inline kickthemout.py]: `kickwhitelistoff ` --> `kickexceptoff`

**Comment by DevAlone** [comment]
So, will it be merged?

**Comment by k4m4** [comment]
> So, will it be merged?

I am waiting for @hamperfait to complete the requested changes in the above review.

**Comment by DevAlone** [comment]
It's been a year, you're very patient :)

**Comment by hamperfait** [comment]
Bummer, I'm really sorry, I totally forgot when I received the request changes until now, and I've never been a big commiter, so I was a tad lost about how to accept those changes.
Hopefully I did it right.

**Pull request closed without merging**